graders/python/requirements.txt

```diff
@@ -12,17 +12,17 @@ networkx==3.4.2
 nltk==3.9.1
 numpy==1.26.4
 openpyxl==3.1.5
-pandas==2.3.0
+pandas==2.3.1
 Pillow==11.3.0
-PuLP==3.2.1
+PuLP==3.2.2
 Pygments==2.19.2
-pyarrow==20.0.0
+pyarrow==21.0.0
```

Comment:
Seems like the python changes are just bug fixes? https://arrow.apache.org/release/21.0.0.html

Going to cross my fingers that no instructor out there is using this to do anything crazy.